src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  height: 408px;
+  padding-top: 32px;
+
+  border: 1px solid $border-color;
+
+  &__container {
+    padding: 0 16px 16px;
+  }
+
+  &__img {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+
+    img {
```

Comment:
Don't use tag names for styling (except html and body)
<img width="856" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/63243136/2605ae26-8754-4ff5-95a6-7af5b8c3c4f2">